amadeus-streaming/src/distinct.rs

```diff
@@ -332,6 +333,94 @@ where
 		});
 	}
 
+	/// Returns a serialized representation of this HHL directly by BigQuery.
+	/// See [Zetasketch from Google](https://github.com/google/zetasketch) for more details on the serialization format.
+	/// See [HLL functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/hll_functions) in BigQuery
+	/// to understand how to store and use the sketches generated by this function.
+	/// Note: this serialization  function doesn't support the sparse format as defined in Zetasketch mostly because
+	/// this HyperLogLogPlusPlus implementation doesn't support it.
+	pub fn serialize_as_proto(&self) -> Vec<u8> {
+		let mut buf = Vec::new();
+		let mut stream = CodedOutputStream::new(&mut buf);
+
+		// We use the NoTag write methods for consistency with the parsing functions and for
+		// consistency with the variable-length writes where we can't use any convenience function.
+		stream
+			.write_uint32_no_tag(crate::proto_util::TYPE_TAG)
+			.expect("proto serialization failed");
```

Comment:
The repeated expect calls are quite noisy here. What's the likelihood of hitting them? I can't imagine a circumstance that would hit them rather than say the Vec triggering an OOM. If so then `.unwrap()` might be worth using instead for easier to read code.

Reply:
Done